packages/fuel-indexer-macros/src/indexer.rs

```diff
@@ -449,10 +460,25 @@ fn process_fn_items(
 
                     for receipt in tx.receipts {
                         match receipt {
-                            Receipt::Call { param1, to: id, ..} => {
+                            Receipt::Call { id: contract_id, amount, asset_id, gas, param1, to: id, .. } => {
                                 #contract
+                                fn selector_to_fn_name(sel: u64) -> String {
```

Comment:
Maybe we could move this to the decoder? Since that's where all the other match logic is?